tutorials/notebook.md

```diff
@@ -0,0 +1,569 @@
+---
+layout: default
+title: Notebook
+parent: Tutorials
+nav_order: 2
+---
+
+# Building a Notebook
+We're going to build a command-line timestamped note-taking application using
+the [Tupelo.js](https://github.com/QuorumControl/tupelo.js "Tupelo.js") API.
+When we're done, we'll be able to use this app to save small timestamped notes
+over time, and later display all the notes in order.
+
+## Getting Started
+Our notebook will be a [Node.js](https://nodejs.org/en/ "Node.js") application.
+We'll manage it's dependencies with [NPM](https://www.npmjs.com/ "NPM"), so make
+sure you have already installed it before going further.
+
+Next, we'll create and enter the directory that will hold our notebook
+application's code, and then initialize a new NPM application with `npm init`.
+The `npm init` defaults are fine for our purposes, so we'll use the `-y` option
+to skip any confirmation questions.
+
+```bash
+# Create the 'notebook' application directory
+mkdir notebook
+
+# Change to the application directory
+cd notebook
+
+# Initialize a new npm application
+npm init -y
+```
+
+This gives us a new directory with a `package.json` skeleton to start with.
+
+## Running the Tupelo RPC server
+The Node.js client cannot directly manage chain trees or connect to the notary
+group, so node applications must instead proxy through an RPC server to work
+with Tupelo.
+
+To install the server, first contact us to get a Tupelo binary for your platform
+and save it within your command `PATH` variable. If you do not wish to save the
+binary in your `PATH`, you can still execute it with the fully qualified or
+relative path to your chosen location for the binary.
+
+After successfully installing the binary, you can run the RPC server by invoking
+`tupelo` (or the path to your chosen install location) along with the necessary
+options. Production applications will usually connect to an independent notary
```

Comment:
the global notary group

Reply:
Is this really the "global" notary group? I can see a day when we would be running multiple test networks, and people can also theoretically set up their own private notary groups as well.

Reply:
oops - never followed up on this. I think in the future you're correct, but for now it's easier to tell people that they are connecting to "the" notary group.